jsettlers.main.swing/src/jsettlers/main/swing/SwingManagedJSettlers.java

```diff
@@ -79,6 +81,44 @@ public static void main(String[] args) throws FileNotFoundException, IOException
 		generateContent(argsMap, content);
 	}
 
+	private static String getSelectedFolderName() {
+		
+		//- Info: Don't use JFileChooser(), some Windows User have problems caused by a NullPointerException in FileDialog()
+		//-   see issue #240: https://github.com/jsettlers/settlers-remake/issues/240
+		
+		
+		//- see: http://stackoverflow.com/questions/1224714/how-can-i-make-a-java-filedialog-accept-directories-as-its-filetype-in-os-x
+		JFrame frame = new JFrame();
+		
+		System.setProperty("apple.awt.fileDialogForDirectories", "true");
```

Comment:
# apple.awt : I know...this is the recommendation on Stackoverflow.
# Select folder on windows: yes, I know. The default Folder-Selection-common-Dialog is not available with Java (only with the org.eclipse.swt.* pack) so may we change the title of this dialog to "selet the file x.y"
# alternative:
1. stay with JFileChooser() and tell users with the bug to change the config by hand
2. use JFileChooser() with a fallback to FileDialog()
3. create ouer own directorychooser
4. use the (org.eclipse.swt.*) Dir File Selection